detox/src/client/Client.js

```diff
@@ -63,8 +63,12 @@ class Client {
     return this._serverUrl;
   }
 
+  async open() {
+    return this._asyncWebSocket.open();
+  }
```

Comment:
@noomorph please let me know if you are comfortable with this change.
Apparently, when introducing the pluggable drivers, they made a hidden assumption that the web socket is available to the outside world. I'm highly against keeping it that way, yet suppose we ought to make the Client useable from within the driver, nonetheless.